src/views/Home.vue

```diff
@@ -25,6 +25,14 @@
           unelevated
           >書き出し</mcw-button
         >
+
+        <mcw-button @click="saveProjectFile" :disabled="uiLocked" unelevated
+          >プロジェクト保存</mcw-button
+        >
+
+        <mcw-button @click="loadProjectFile" :disabled="uiLocked" unelevated
+          >プロジェクト読み込み</mcw-button
```

Comment:
UI側の都合ですが、保存と読み込みの字面が非対称に見えそうです。
ちょっと統一感を持たせるために「読込」にするのはどうでしょうか。
```suggestion
          >プロジェクト読込</mcw-button
```